cli/man/splinter-authid.1.md

```diff
@@ -48,11 +48,14 @@ SUBCOMMANDS
 : Shows a specific authorized identity on a Splinter node
 `create`
 : Creates an authorized identity on a Splinter node.
+`update`
+: Updates an existing authorized identity on a Splinter node.
 
 SEE ALSO
 ========
 | `splinter-authid-list(1)`
 | `splinter-authid-show(1)`
 | `splinter-authid-create(1)`
```

Comment:
These should be alphabetical

Reply:
fixed; moved to the first commit where this file was introduced